scram-package-monitor-sender.py

```diff
@@ -33,7 +33,7 @@ def esReportPackages(results):
     except HTTPError as e:
         print(e)
         try:
-            print(result.read())
+            print(result.read().decode())
```

Comment:
@iarspider , checkif ` base64.encodestring` works in python3 ... I think it is deprecated

Reply:
Works but deprecated since 3.1 .